x-pack/solutions/search/plugins/search_playground/common/models.ts

```diff
@@ -46,8 +46,8 @@ export const MODELS: ModelProvider[] = [
     provider: LLMs.bedrock,
   },
   {
-    name: 'Google Gemini 1.5 Pro',
-    model: 'gemini-1.5-pro-002',
+    name: 'Google Gemini 2.5 Pro',
```

Comment:
@joemcelroy do you think we should add the 2.5 models in addition the 1.5 or replace 1.5 with 2.5?

Reply:
nm, it looks like 1.5 is being discontinued next month, so replace seems right.